paparazzi/settings.gradle

```diff
@@ -1,5 +1,3 @@
-rootProject.name = 'paparazzi-libs'
```

Comment:
> Project accessors enabled, but root project name not explicitly set for 'paparazzi'. Checking out the project in different folders will impact the generated code and implicitly the buildscript classpath, breaking caching.

Reply:
Even if you manage to use `paparazzi` as the name, make it explicit.

Reply:
Agreed. I'll try. Thanks!

Reply:
Well, no good news on the Gradle front... it is simply not possible: https://github.com/gradle/gradle/issues/16608

Reply:
Interesting 🤔 

Is the solution to rename the `paparazzi` directory to `paparazzi-libs`? Alternatively, all the modules under `paparazzi/` could be moved to the root directory.

Reply:
It needs to be in a subdir, so that the top level can include it. Rename might work. OR rename :paparazzi to something else, but that would affect artifact names I guess or make the dir-artifact naming inconsistent.

Reply:
Due to the above, it'll never compile with name = paparazzi :(. Not even in IDEA sync/run, because that just executes Gradle. With your recent change CI will be green, but locally it should fail sync/build/run.

Reply:
Indeed. This is a tricky issue 😅. I'll keep trying though!

Reply:
Perhaps the only way is to add a system gradle property that updates the root project name for local development. I was hoping the experience would be more seamless though.

Reply:
It doesn't matter how you set it, this "active" thing is also a property. Did you try syncing with the flag, does it work locally?

Reply:
Actually, after checking this a little closer, I was able to build with these changes from both IntelliJ and Android Studio. Maybe this is working as intended?